articles/TOC.yml

```diff
@@ -1,44 +1,51 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Getting started
   items:
-  - name: Get started
-    href: welcome.md
-  - name: Five questions about quantum computing
-    items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
+  - name: How can I learn more?
+    href: overview/how-to-learn-quantum-computing.md
+- name: The QDK introduction
+  href: welcome.md
   items:
-  - name: Install the QDK
+  - name: Install and update
     items:
-    - name: Installation overview
-      href: install-guide/index.md
-    - name: Q# + C#
-      href: install-guide/csinstall.md
-    - name: Q# + Python
-      href: install-guide/pyinstall.md
-    - name: Q# Jupyter notebooks
-      href: install-guide/qjupyter.md
-  - name: Update the QDK
-    href: install-guide/update.md  
-- name: Quickstarts
+    - name: Install the QDK
+      items:
+      - name: Installation overview
+        href: install-guide/index.md
+      - name: Q# + C#
+        href: install-guide/csinstall.md
+      - name: Q# + Python
+        href: install-guide/pyinstall.md
+      - name: Q# Jupyter notebooks
+        href: install-guide/qjupyter.md
+    - name: Update the QDK
```

Comment:
Can the indentation change for "Update the QDK" and "How to create a Q# project" to align with the arrow of "Install the QDK"? Visually people may thing the lower ones are under the upper one with the arrow and may miss them.

![image](https://user-images.githubusercontent.com/59890648/73984766-47aa9780-48ee-11ea-8f3d-b85b12df9eed.png)

Reply:
Unfortunately I'd guess that's something on the far, far-backend---probably out of reach for tuning within individual doc sets. For example, this page has the identical behavior: https://docs.microsoft.com/en-us/dotnet/core/